app/manifest.cue

```diff
@@ -83,9 +83,9 @@ appManifest: versions: v1alpha1: {
 			kinds: [...#ManifestKind]
 			// ExtraPermissions contains additional permissions needed for an app's backend component to operate.
 			// Apps implicitly have all permissions for kinds they managed (defined in `kinds`).
-			extraPermissions: {
+			extraPermissions?: {
 				// accessKinds is a list of KindPermission objects for accessing additional kinds provided by other apps
-				accessKinds: [...#KindPermission]
+				accessKinds?: [...#KindPermission]
```

Comment:
I think this one shouldn't be optional, right? Otherwise we can have `extraPermissions` with empty content, which doesn't make much sense.

Reply:
Good point, I'll take that off